packages/slate/src/models/range.js

```diff
@@ -599,7 +616,8 @@ class Range extends Record(DEFAULTS) {
    */
 
   moveToAnchor() {
-    return this.moveFocusTo(this.anchorKey, this.anchorOffset)
+    const { anchorPoint: focusPoint } = this
```

Comment:
This is a kind of confusing notation. It would be better to make these just use `focus: this.anchor` instead of the tricky renaming.